src/Tendermint/Signer.h

```diff
@@ -0,0 +1,48 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "../proto/Cosmos.pb.h"
+#include "../Data.h"
+#include <nlohmann/json.hpp>
+#include <stdint.h>
+#include <vector>
+
+using json = nlohmann::json;
```

Comment:
Don't put using directives in header files because they contaminate the global scope.

Reply:
> Don't put using directives in header files because they contaminate the global scope.

done